templates/Sutil-Safe/Build.fs

```diff
@@ -39,10 +39,12 @@ module Helpers =
 module Paths =
   let deploy = Path.getFullName "deploy"
   let client = Path.getFullName "src/Client"
+  let publish = Path.getFullName "src/Client/public"
```

Comment:
I like `build` or `output` a little bit better than `publish`, due to `publish` being a dotnet command and its output being generally redirected to a `publish` dir. 
As for the path, I wouldn't necessarily merge the build output with the public directory, I think they serve different purposes, since we're still running the output through webpack.
What do you think about following?
```suggestion
  let output = Path.getFullName "src/Client/output"
```